nvflare/fuel/hci/client/fl_admin_api.py

```diff
@@ -521,6 +532,34 @@ def set_timeout(self, timeout: float) -> FLAdminAPIResponse:
             APIStatus.ERROR_RUNTIME, {"message": "Runtime error: could not handle server reply."}, reply
         )
 
+    @wrap_with_return_exception_responses
+    def list_sp(self) -> FLAdminAPIResponse:
+        success, reply_data_full_response, reply = self._get_processed_cmd_reply_data("list_sp")
+        if reply.get("data"):
+            return FLAdminAPIResponse(APIStatus.SUCCESS, reply.get("data"), reply)
```

Comment:
Just curious on why this is using the "data" to carry the information, rather than "details" as the other commands.

Reply:
"details" contains the str to display for the cli, but "data" is the original object so the user of the api can more easily access the contents.